components/email.tsx

```diff
@@ -31,6 +31,12 @@ const Email = () => {
       )
       setPassword('')
     }
+    else if (password === 'tastyencoding') {
```

Comment:
I suggest hashing the password so that people can’t find it by looking at the code 👁️👁️

or maybe we want people to look at the code? What do you think?

Reply:
The other password isn't too hidden either, but given the added challenge of this one hashing would certainly add a reasonable layer to it. 
Not sure how I'd exactly implement that though, or if it's fully worth the time; do we want to add client-side hashing to the website for a gag password check?

Reply:
Hashing would be super easy because there’s a library on npm that does it for you. But I agree that it’s probably not worth adding a whole new library just for that. Plus it’s kind of fun for people to dig through the code and find an Easter egg. Merging!!!